app/token/token.go

```diff
@@ -1,3 +1,4 @@
+// Package token
```

Comment:
what the difference with `app/token/abstract.go` which is `// Package token provides a way to serialize/un-serialize data structures in an encrypted token.`?

Reply:
fixed